R/r.R

```diff
@@ -8,6 +8,7 @@
 #'   Packages](https://r-pkgs.org).
 #' @export
 use_r <- function(name = NULL) {
+  check_file_name_length(name)
```

Comment:
I think it would be pretty self-consistent of us to do something simpler in this case. I propose `stopifnot(is_string(name))` instead of a custom helper.

Reply:
Also, the check should happen after the `name <- name %||% ...` line, because `NULL` input is valid.

Reply:
I agree all except that we use `stopifnot()`. A regular if-not-true-stop branch might be more useful to give better error messages for the end-users.

Reply:
On the other hand, I see that `stopifnot(is_string(name))` is often used in the package to check inputs.